spec/callAsyncSpec.js

```diff
@@ -0,0 +1,7 @@
+const callAsync = require('../src/callAsync');
+
+describe('callAsync', function() {
+  it('lets you handle callback results using promises', async function() {
+    expect(await callAsync((done) => process.nextTick(done, null, 'hello'))).toBe('hello');
```

Comment:
If we were using jest we could
```suggestion
    await expect(callAsync((done) => process.nextTick(done, null, 'hello'))).resolves.toBe('hello');
```